packages/agents/src/scriptedAgents/DataAnalyst/DataAnalyst.spec.ts

```diff
@@ -185,4 +185,54 @@ Fern,Green
 34,102,Alice,Doctor,120000
 45,103,Bob,Lawyer,95000`);
   });
+
+  test("AnswerQuestionSmallCsv", async () => {
+    const { agent, debugLog } = createDataAnalystAgent(
+      "answer-question-small-csv",
+      [path.join(__dirname, "testInputs/answerQuestionSmallCsv/file1.csv")]
+    );
+    const response = await runDataAnalystAgent(
+      agent,
+      "How much was spent on utilities in total ? Write the answer in an output.txt file.",
+      debugLog
+    );
+
+    expect(response.value.ok).toBe(true);
+    const result = agent.workspace.readFileSync("output.txt");
+    expect(result).toContain("84.35");
```

Comment:
```suggestion
    expect(result).toContain("84");
```
It's okay if the answer just contains "84"

Reply:
that's the way we guarantee that we assert as the benchmark expects, so i think having the decimals is good

Reply:
The benchmark only requires it to contain "84"

https://github.com/Significant-Gravitas/AutoGPT/blob/master/benchmark/agbenchmark/challenges/verticals/data/4_answer_question_small_csv/data.json

Reply:
right, my bad!